Spanner/src/Connection/Grpc.php

```diff
@@ -120,6 +122,16 @@ class Grpc implements ConnectionInterface
      */
     private $longRunningGrpcClients;
 
+    private function enableConnectionManagement($conf_path)
+    {
+        // TODO(ddyihai): move this function to GrpcTrait, if we are going
+        // to enable the grpc-gcp library for other apis.
+        $conf = new \Grpc\Gcp\ApiConfig();
```

Comment:
Could we add a `use` statement at the top of the file for this class?

Reply:
Done.